srfi-alist.html

```diff
@@ -66,6 +70,442 @@ <h2 id="rationale">Rationale</h2>
+            <p>Guile</p>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <p><strong>sloppy-assv</strong></p>
+          </td>
+          <td>
+            <p>key alist</p>
+          </td>
+          <td>
+            <p>Guile</p>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    <h4>Get existing entry by value instead of key</h4>
+    <table>
+      <thead>
+        <tr>
+          <th>Procedure</th>
+          <th>Signature</th>
+          <th>Provided by</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>
+            <p><strong>reverse-assoc</strong></p>
+          </td>
+          <td>
+            <p>obj alist [=]</p>
+          </td>
+          <td>
+            <p>Cowan</p>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <p><strong>reverse-assq</strong></p>
+          </td>
+          <td>
+            <p>obj alist</p>
+          </td>
+          <td>
+            <p>Cowan</p>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <p><strong>reverse-assv</strong></p>
+          </td>
+          <td>
+            <p>obj alist</p>
+          </td>
+          <td>
+            <p>Cowan</p>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    <h4>Traverse list</h4>
+    <table>
+      <thead>
+        <tr>
+          <th>Procedure</th>
+          <th>Signature</th>
+          <th>Provided by</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>
+            <p><strong>alist-for-each</strong></p>
+          </td>
+          <td>
+            <p>proc alist</p>
+          </td>
+          <td>
+            <p>SLIB</p>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <p><strong>alist-map</strong></p>
+          </td>
+          <td>
+            <p>proc alist</p>
+          </td>
+          <td>
+            <p>SLIB</p>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+
+    <h4>Abstract list</h4>
+    <table>
+      <thead>
+        <tr>
+          <th>Procedure</th>
+          <th>Signature</th>
+          <th>Provided by</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>
+            <p><strong>alist-associator</strong></p>
+          </td>
+          <td>
+            <p>pred</p>
+          </td>
+          <td>
+            <p>SLIB</p>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <p><strong>alist-inquirer</strong></p>
+          </td>
+          <td>
+            <p>pred</p>
+          </td>
+          <td>
+            <p>SLIB</p>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <p><strong>alist-remover</strong></p>
+          </td>
+          <td>
+            <p>pred</p>
+          </td>
+          <td>
+            <p>SLIB</p>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <p><strong>predicate→asso</strong></p>
```

Comment:
The arrow looks strange.

Reply:
Ah, that's from the AsciiDoc converter which turns `->` arrows into Unicode arrows by default. I'll fix it.